### PR TITLE
Test(web,web-react): Update visual snapshots after demo apps redesign #DS-1479

### PR DESCRIPTION
**Pull request by adamkudrna**
Hide site header and footer from component snapshots.

## Description

Component snapshots in general have not changed because, after hiding demo header and footer, the output is exactly the same as it was before the docs redesign (#1666, #1675).

Though, some component snapshots have been updated because docs components now utilize theme design tokens. This includes:

- `EmptyState` (docs-Placeholder),
- `Flex` (docs-Box),
- `Grid` (docs-Box),
- `ScrollView` (docs-Box).

### Issue reference

https://jira.almacareer.tech/browse/DS-1479

### CODE REVIEW

File: tests/e2e/demo-components-compare.spec.ts [hunk 1]
```diff
@@ -37,6 +37,7 @@ const runComponentCompareTests = (testConfig: TestConfig) => {
             const url = getServerUrl(packageName);
             await page.goto(`${url}${componentsDir}/${component}/${packageName === 'web-twig' ? '?HIDE_TOOLBAR' : ''}`);
             await waitForPageLoad(page);
+            await hideFromVisualTests(page);
```
Comment (crishpeen) [inline tests/e2e/demo-components-compare.spec.ts:40]: we should do this in modal tests too.

File: tests/e2e/demo-homepages.spec.ts [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import { isTesting } from '@lmc-eu/spirit-common/constants/environments';
 import { SERVERS, getDevelopmentEndpointUri } from '@lmc-eu/spirit-common/constants/servers';
 import { test } from '@playwright/test';
-import { takeScreenshot, waitForPageLoad } from '../helpers';
+import { hideFromVisualTests, takeScreenshot, waitForPageLoad } from '../helpers';
```
Comment (crishpeen) [inline tests/e2e/demo-homepages.spec.ts:4]: it hides something? 🤔

**Review by literat — APPROVED** [review]
LGTM 👍

**Review by pavelklibani — APPROVED** [review]

File: tests/e2e/demo-homepages.spec.ts (hunk 2 above)
Comment (adamkudrna) [reply tests/e2e/demo-homepages.spec.ts:4]: It's bound to a CSS helper, I cannot tell what exactly it hides…

**Pull request merged**